src/Microsoft.DotNet.ProjectModel/FileSystemGlobbing/HashCodeCombiner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Fine with me.  As long as the dependency to the ASP.NET project is gone, I'm fine.